conf/ref_databases.config

```diff
@@ -130,6 +130,14 @@ params {
             dbversion = "SBDI-GTDB-R07-RS207-1 (https://scilifelab.figshare.com/articles/dataset/SBDI_Sativa_curated_16S_GTDB_database/14869077/4)"
```

Comment:
Key `'sbdi-gtdb'` still leads to Release `R07-RS207-1`, wouldnt it be good to update it?

Reply:
Good catch! I've pushed an update.